hexa/files/api.py

```diff
@@ -170,3 +173,22 @@ def generate_upload_url(bucket_name: str, target_key: str, content_type: str):
     return blob.generate_signed_url(
         expiration=3600, version="v4", method="PUT", content_type=content_type
     )
+
+
+def load_sample_data(bucket_name: str):
+    """
+    Init bucket with default content
+    """
+    static_files_dir = join(dirname(__file__), "static")
```

Comment:
@pvanliefland Isn't there a better way to do this ?

Reply:
Not really. You could use `pathlib` but it's not shorter or easier.